### PR TITLE
starting

### PR DESCRIPTION
**Pull request by prihit**

### CODE REVIEW

**Comment by chayan007** [comment]
@prihit did you forget to push the rest of the code ?

File: templates/v2/layouts/home.html [hunk 1]
```diff
@@ -292,7 +292,11 @@ <h4 class="sub-header-01">Psychometric Test</h4>
           <p>
             We ease your biggest doubts with our AI based technology in finding your dream college.
           </p>
+          {% if not user.is_authenticated %}
           <button class="button-primary" id="psychoid">Get Started</button>
+          {% else %}
+          <div class="start_btn"><button class="button-primary">Get Started</button></div>
```
Comment (chayan007) [inline templates/v2/layouts/home.html:298]: Where is the modal for this ?

File: templates/v2/layouts/home.html [hunk 2]
```diff
@@ -292,7 +292,11 @@ <h4 class="sub-header-01">Psychometric Test</h4>
           <p>
             We ease your biggest doubts with our AI based technology in finding your dream college.
           </p>
+          {% if not user.is_authenticated %}
           <button class="button-primary" id="psychoid">Get Started</button>
```
Comment (chayan007) [inline templates/v2/layouts/home.html:296]: Unauthenticated should prompt user to login.

File: templates/v2/layouts/home.html (hunk 1 above)
Comment (prihit) [reply templates/v2/layouts/home.html:298]: I have not added it yet as this button was also redirecting to login modal

File: templates/v2/layouts/home.html (hunk 2 above)
Comment (prihit) [reply templates/v2/layouts/home.html:296]: but else condition is not working even

**Review by chayan007 — APPROVED** [review]

**Pull request merged**